### PR TITLE
Update GCP doc - setting a custom domain

### PR DESCRIPTION
**Pull request by BAlmeidaS**
Changing the section *Using Your Own Domain* to be compliant with Kubeflow 1.1 version using kpt.

### CODE REVIEW

File: content/en/docs/gke/custom-domain.md [hunk 1]
```diff
@@ -22,38 +15,24 @@ so, follow the guide to
 
 If you want to use your own domain instead of **${KF_NAME}.endpoints.${PROJECT}.cloud.goog**, follow these instructions after running `kfctl build`:
 
-1. Replace `hostname` in `kustomize/iap-ingress/base/params.env` with your own domain.
+1. Remove the substitution `hostname` in kptfile.
```
Comment (8bitmp3) [inline content/en/docs/gke/custom-domain.md:18]: Suggestion: 

- "kptfile" -> "the Kptfile" (reference: https://googlecontainertools.github.io/kpt/search/?q=kptfile)

```suggestion
1. Remove the substitution `hostname` in the Kptfile.
```

File: content/en/docs/gke/custom-domain.md [hunk 2]
```diff
@@ -22,38 +15,24 @@ so, follow the guide to
...
-    oauthSecretName=kubeflow-oauth
-    ...
+    kpt cfg delete-subst instance hostname
     ```
 
-1. Apply the changes.
-
-    Using `kfctl`.
-
+1. Create a new setter `hostname` in kptfile.
```
Comment (8bitmp3) [inline content/en/docs/gke/custom-domain.md:23]:
```suggestion
1. Create a new setter `hostname` in the Kptfile.
```

File: content/en/docs/gke/custom-domain.md [hunk 3]
```diff
@@ -64,7 +43,7 @@ If you want to use your own domain instead of **${KF_NAME}.endpoints.${PROJECT}.
 1. Get the address of the static IP address created.
     ```
```
Comment (8bitmp3) [inline content/en/docs/gke/custom-domain.md:44]: Suggestion:

- Add space before the code block

```suggestion

    ```
```

File: content/en/docs/gke/custom-domain.md [hunk 4]
```diff
@@ -22,38 +15,24 @@ so, follow the guide to
...
-    ```
-
-    Delete the endpoint created by the `cloud-endpoints-controller`, if any.
+1. Apply the changes.
     ```
-    gcloud endpoints services delete ${KF_NAME}.endpoints.${PROJECT}.cloud.goog
+    make apply-kubeflow
     ```
 
 1. Check Ingress to verify that your domain was properly configured.
```
Comment (8bitmp3) [inline content/en/docs/gke/custom-domain.md:38]: Suggestion:

- Add space before the code block after the instruction step

```suggestion
1. Check Ingress to verify that your domain was properly configured.

```

**Review by 8bitmp3 — CHANGES_REQUESTED** [review]
Hey @BAlmeidaS Thanks for the update.

A few suggestions: 

- "kptfile" -> "the Kptfile" (reference: https://googlecontainertools.github.io/kpt/search/?q=kptfile)

```
1. Remove the substitution `hostname` in the Kptfile.
...
1. Create a new setter `hostname` in kptfile.
```

Also, can you please add spaces before a couple of code blocks? (Better formatting, as per the Google style guide).

LMKWYT! Assigning the GCP team

/assign @Bobgy @joeliedtke @rmgogogo

cc @RFMVasconcelos (regarding outdated docs)

**Comment by BAlmeidaS** [comment]
I applied your suggestions @8bitmp3. Tks for your review

**Comment by Bobgy** [comment]
/assign @jlewi 
Can you help review?

Looks good for me, but I don't have much knowledge on how to configure custom domain name.
/approve
/hold

**Comment by 8bitmp3** [comment]
gently ping @jlewi

**Comment by jlewi** [comment]
/lgtm
/approve

Thanks

**Pull request merged**